app/models/invenio_rdm_record_converter.rb

```diff
@@ -226,6 +219,18 @@ def build_creator_contributor_json(creator)
     File.write(MEMOIZED_PERSON_OR_ORG_DATA_FILE, @@person_or_org_data)
     # return the actual json
     json
+  end # build_creator_contributor_json
+
+  def additional_titles(titles)
```

Comment:
This function is fine as it is, but here's another take for you to consider. I'll leave it up to you if you'd like to change anything or not. 

```
def additional_titles(additional_titles)
  return nil if additional_titles.blank?
  additional_titles.map{ |title| {"title": title, "type": "alternative_title", "lang": ENG} }
end

additional_title(titles.shift)
```